src/main/java/io/github/devatherock/json/formatter/JSONFormatter.java

```diff
@@ -57,12 +58,66 @@ protected boolean removeEldestEntry(Entry<Integer, String> eldest) {
 		}
 	};
 
+	private boolean useSlf4jLevelNames = false;
+
+	public JSONFormatter() {
+		configure();
+	}
+
+	/**
+	 * Configure a {@link JSONFormatter} from LogManager properties.
+	 */
+	private void configure() {
+		LogManager manager = LogManager.getLogManager();
+		String cname = getClass().getName();
+
+		String value = manager.getProperty(cname + ".use_slf4j_level_names");
+		useSlf4jLevelNames = Boolean.valueOf(value);
+
+		value = manager.getProperty(cname + ".key_timestamp");
+		if (value != null) {
+			Constants.KEY_TIMESTAMP = value;

```

Comment:
Now that they are not constants anymore, maybe we can move the fields that are configurable out of the `Constants` class and have them as instance level variables similar to `useSlf4jLevelNames`, with instance variable naming conventions like `timestampKey`, `loggerNameKey` etc and the default values as whatever values they have now.

Reply:
https://github.com/devatherock/jul-jsonformatter/pull/8